woocommerce/wc-admin
# Use 'H' component for chart header

Coming from https://github.com/woocommerce/wc-admin/issues/208#issuecomment-414408391.

Replaces the chart header element (which was a `span`) with a semantic header.

**Steps to test**
- Go to any page with a chart.
- Inspect the title item (with class `.woocommerce-chart__title`) and verify it's a header HTML element (`h1`, `h2`, `h3`..).
